libnavui-maps/src/test/java/com/mapbox/navigation/ui/maps/internal/route/line/MapboxRouteLineUtilsTest.kt

```diff
@@ -990,7 +615,7 @@ class MapboxRouteLineUtilsTest {
             listOf("motorway")
         )
 
-        assertTrue(result.all { it.segmentColor == -1 })
+        Assert.assertTrue(result.all { it.segmentColor == -1 })
```

Comment:
NIT

```suggestion
       assertTrue(result.all { it.segmentColor == -1 })
```

Same thing happens with the rest of the tests below 👇